app/components/lock_old_posts.py

```diff
@@ -0,0 +1,54 @@
+import datetime as dt
+
+import discord
+
+from app import config
+from app.utils import dynamic_timestamp, post_is_solved
+
+
+async def check_for_old_posts(message: discord.Message) -> None:
+    post = message.channel
+    now = dt.datetime.now(tz=dt.UTC)
+    two_fortnights_ago = now - dt.timedelta(weeks=4)
```

Comment:
Can we please make this something like this instead? 😵‍💫 
```suggestion
    stale_threshold = now - dt.timedelta(days=30)
```

Reply:
Lol fine 😅.